custom_components/prometheus_sensor/sensor.py

```diff
@@ -30,15 +30,17 @@
 CONF_QUERIES = "queries"
 CONF_EXPR = "expr"
 CONF_STATE_CLASS = "state_class"
+CONF_UNIQUE_ID = "unique_id"
```

Comment:
We can reuse the definition from `homeassistant.const`.

https://github.com/home-assistant/core/blob/3f11fbfda866109307b3a8cd62bd5e7621a78f78/homeassistant/const.py#L253

Reply:
Thanks! I've updated the PR to reuse the constant